flask_rebar/rebar.py

```diff
@@ -102,6 +103,7 @@ def _wrap_handler(
     request_body_schema=None,
     headers_schema=None,
     marshal_schema=None,
+    produces=None,
```

Comment:
I'm not a fan of this interface.

Its conflated with "produces" from OpenAPI 2.0.

Also, this doesn't actually guarantee that the handler actually returns the content-type specified here. It just patches the headers.

So maybe this should be a "headers" parameter, with extra headers to set on the response? That seems more flexible and more clear.